apps/ewallet/lib/ewallet/seeders/cli_seeder.ex

```diff
@@ -43,8 +43,8 @@ defmodule EWallet.Seeder.CLI do
       |> Seeder.argsline_for()
       |> process_argsline()
 
-    IO.puts("\n-----\n")
-    IO.gets(@confirm_message)
+    _ = IO.puts("\n-----\n")
```

Comment:
Interesting! Does each function call should be patterned matched in this way?

Reply:
only when you wan't to ignore the returned value